app/loader.h

```diff
@@ -68,6 +70,8 @@ class Loader: public QObject
     QgsProject *mProject = nullptr;
     QgsQuickPositionKit *mPositionKit = nullptr;
     bool mRecording = false;
+    //! Exists only while loading project with non-empty path
+    QString mLoadingFile = QString( "%1/.input_loading_project" ).arg( QStandardPaths::standardLocations( QStandardPaths::TempLocation ).first() );
```

Comment:
move comment to function "loadingFile" . Also I do not think we need new member variable, path can be created on the fly

Reply:
What about make it static (const), so it can be also accessible from main.cpp (the existence of the file is checked there on init) and it will be created only once?

Reply:
sounds good...